### ISSUE TITLE
Weirdly stretched settings list items bug

### ISSUE THREAD

**Issue by Snow4DV** [dessalines/jerboa#1338]
### Jerboa Version

0.0.55 debug / main branch (8b5c6fe)

### Android Version + Phone

Pixel 4 XL API 28/34 (physical device & emulator)

### Describe The Bug

By some reason menu items are stretched like this in the latest debug build
![image](https://github.com/dessalines/jerboa/assets/32912696/b4e28391-d003-44f3-965f-a49b4a60c571)

### To Reproduce

1. Build app using latest commit ([8b5c6fe](https://github.com/dessalines/jerboa/commit/8b5c6fe24fda700b294781078187a5797eafc42d))
2. Open the app
3. Go to settings

**Comment by Snow4DV** [comment]
The issue is caused by this commit: [9e1a79da (Upgrading deps)](https://github.com/dessalines/jerboa/commit/9e1a79da633d6644d8245b91f6d94e29960c507b) 

Probably something broke the SettingsMenuLink composable that comes from aloma library even though it wasn't updated

**Comment by MV-GH** [comment]
This will definitely need to be fixed before we do any release

**Comment by dessalines** [comment]
Definitely. This probably has to do with the newer materialv3 RC in that commit.

**Comment by Snow4DV** [comment]
@dessalines 

Oops, looks like you skipped my SettingsListDropdown with action preset when merging temp workaround (#1345)

**Comment by dessalines** [comment]
Oops, my bad. I have a PR open on that compose-settings repo also.

**Comment by MV-GH** [comment]
Ironically if the downmigrations worked this wouldn't be a problem. Yeah you can just push a new release that includes the setting

**Comment by Snow4DV** [comment]
> Ironically if the downmigrations worked this wouldn't be a problem. Yeah you can just push a new release that includes the setting

Everything is in place, setting too except the height modifier for it (only swipe preset setting is stretched) so no reason to care much about it. Just a visual glitch 

![Screenshot_20240206-004917](https://github.com/dessalines/jerboa/assets/32912696/c5ed6cfd-b967-40c0-8cee-5777d3ed4202)

**Comment by dessalines** [comment]
I'll get this one in a sec.

This one's also a bigger issue : https://github.com/dessalines/jerboa/issues/1348
